pytorch_lightning/core/step_result.py

```diff
@@ -576,7 +576,7 @@ def get_non_metrics_keys(self):
         """
         This function is used to filter metric keys for which the value isn't a Metric
         """
-        return [k for k, v in self.items() if not isinstance(v, Metric)]
+        return [k for k, v in self.items() if (not isinstance(v, Metric) and k != 'meta')]
```

Comment:
could you describe what changes now that `meta` is excluded?

Reply:
The meta is an internal key `self['meta'] = {'_internal': {'_reduce_on_epoch': False, 'batch_sizes': []}}` and shouldn't be returned.